package.json

```diff
@@ -88,24 +90,36 @@
     "yargs": "^16.0.0"
   },
   "devDependencies": {
-    "@netlify/eslint-config-node": "^3.2.8",
+    "@ava/typescript": "^2.0.0",
+    "@netlify/eslint-config-node": "github:netlify/eslint-config-node#feat/typescript",
```

Comment:
The CI seems to be failing because of this. Would you like to cut a release candidate of `@netlify/eslint-config-node`?

Reply:
Yes, please!

Reply:
Yes, that's a good idea!

Reply:
7f7b607